### PR TITLE
Plone 5.1 Support

### PR DESCRIPTION
**Pull request by maethu**
Noticeable changes:
- Fix init of soup catalog in order to not have a write on read on first access of the soup catalog.
- Add plone 5 profile for bundles
- Fix duplicated entries in activity stream for DX based content types.
- Convert the formlib based portlet to z3c.form. No migration needed, since it only changes the form instances.
- Fix weird behavior with freeze - see readme and 
7e20066

### CODE REVIEW

**Comment by maethu** [comment]
ugh I broke plone 4... gonna fix it later today

**Comment by maethu** [comment]
It is working now @jone 💛

**Comment by maethu** [comment]
The js part on a booted plone 5 does work too

File: ftw/activity/tests/test_tab.py [hunk 1]
```diff
@@ -24,22 +24,22 @@ def test_activity_fetch_url_points_to_activity_tab(self, browser):
...
-            clock.backward(days=1)
-            create(Builder('page').titled('Two'))
-            clock.backward(days=1)
-            create(Builder('page').titled('Three'))
+    # @browsing
+    # def test_render_events(self, browser):
+    #     with freeze(datetime(2010, 1, 2)) as clock:
+    #         create(Builder('page').titled(u'One'))
+    #         clock.backward(days=1)
+    #         create(Builder('page').titled(u'Two'))
```
Comment (jone) [inline ftw/activity/tests/test_tab.py]: Remove or fix?

Comment (maethu) [reply ftw/activity/tests/test_tab.py]: oooops
 I'll fix it

Comment (maethu) [reply ftw/activity/tests/test_tab.py]: @jone It's fixed

**Pull request merged**